src/xdocs/config_filters.xml

```diff
@@ -420,17 +420,34 @@ public class UserService {
     </section>
 
     <section name="SuppressionFilter">
+      <p>Since Checkstyle 3.2</p>
       <subsection name="Description" id="SuppressionFilter_Description">
-        <p>Since Checkstyle 3.2</p>
           <p>
             Filter <code>SuppressionFilter</code> rejects
             audit events for Check errors according to
-            a <a href="dtds/suppressions_1_2.dtd"><em>suppressions XML
-            document</em></a> in a file. If there is no configured
+            a <a href="https://checkstyle.org/dtds/suppressions_1_2.dtd">suppressions XML
+            document</a> in a file. If there is no configured
             suppressions file or the optional is set to true and
             suppressions file was not found the Filter accepts all audit events.
           </p>
       </subsection>
+      <subsection name="Notes" id="SuppressionFilter_Notes">
```

Comment:
please move Notes below Properties section.
As we need to show all list of properties first and only below explain nuances. Please make link from `file` at Properties to Notes, something like "See details at Notes" or ...

Reply:
done

Reply:
> Please make link from file at Properties to Notes, something like "See details at Notes" or ...

not done, but could be done any time after this PR.